operator/hack/lokistack_dev.yaml

```diff
@@ -9,3 +9,7 @@ spec:
     secret:
       name: test
   storageClassName: standard
+  template:
+    ruler:
+      replicas: 1
```

Comment:
change to 2 - so we test what @Red-GV was asking (this will be deployed with the new stateful-set code)